packages/plugin-js/test/build.test.js

```diff
@@ -45,4 +45,23 @@ describe('@cobalt-ui/plugin-js', () => {
       expect(fs.readFileSync(new URL('actual.json', cwd), 'utf8'), `nested: JSON`).toMatchFileSnapshot(fileURLToPath(new URL('want.json', cwd)));
     });
   });
+
+  test('exclude meta from output when options.meta is false', async () => {
+    const cwd = new URL(`./meta/`, import.meta.url);
+    const tokens = JSON.parse(fs.readFileSync(new URL('tokens.json', cwd)));
+    await build(tokens, {
+      outDir: cwd,
+      plugins: [
+        pluginJS({
+          js: 'actual.js',
+          json: 'actual.json',
+          meta: false,
+        }),
+      ],
+      color: {},
```

Comment:
Its unclear why the tests fail without including this line in the call to `build` given that its not required for users to have it in their config files

Reply:
This is the difference between **user config** and **resolved, normalized config.** In userland, we’re lax about config properties, and we fill in the gaps and provide defaults for anything not specified. But internally, we must fill in all defaults and normalize everything. This is a test of the internal build command which doesn’t want ambiguity; it wants all options filled in.